lib/tasks/publishing_api.rake

```diff
@@ -1,46 +1,14 @@
 namespace :publishing_api do
   desc "Publish special routes such as sitemaps"
   task :publish_special_routes do
```

Comment:
Now that rummager's publishing more pages, it would be good to add some unit tests to check that the routes being published are valid against the content schemas. Here's how frontend does it: https://github.com/alphagov/frontend/blob/master/test/unit/special_route_publisher_test.rb